src/main/java/org/synyx/urlaubsverwaltung/statistics/vacationoverview/web/ApplicationForLeaveVacationOverviewViewController.java

```diff
@@ -47,25 +79,217 @@ public String applicationForLeaveVacationOverview() {
 
     @PreAuthorize(IS_PRIVILEGED_USER)
     @GetMapping("/vacationoverview")
-    public String applicationForLeaveVacationOverview(Model model) {
-        Person signedInUser = personService.getSignedInUser();
+    public String applicationForLeaveVacationOverview(
```

Comment:
Ich denke die Methode ist sehr komplex und für einen controller relativ umfassend :)

Reply:
jap, schauen wir uns in eine separaten merge request an.

ich denke die funktionalität hier kann man 1:1 für den kalender auf der startseite nutzen.